### ISSUE TITLE
Loss Functions

### ISSUE THREAD

**Issue by PaperKites** [naoto0804/pytorch-inpainting-with-partial-conv#40]
Hello

Why does the code sum the loss functions together?
https://github.com/naoto0804/pytorch-inpainting-with-partial-conv/blob/e04c84d2bfc2ef8b40ba4af13b08ea43c6057ea8/train.py#L114-L117

I appreciate your help :)

**Comment by xilongzhou** [comment]
Because inpainting loss is the sum of perceptual loss, style loss, valid loss and hole loss. The loss here return the whole inpainting loss every certain iteration.

**Comment by PaperKites** [comment]
@xilongzhou So, one loss function can't represent the inpainting loss.
Could you please mention some academic papers or articles to read more about it? Or keywords to google the subject?

Thank you so much in advance

**Comment by xilongzhou** [comment]
@PaperKites  Here is the paper: Image Inpainting for Irregular Holes Using Partial Convolutions

**Comment by naoto0804** [comment]
@xilongzhou  Thank you for your answer!

**Comment by PaperKites** [comment]
Thank you both
